cli/cmdlineparser.cpp

```diff
@@ -264,19 +264,19 @@ bool CmdLineParser::fillSettingsFromArgs(int argc, const char* const argv[])
         {
             auto it = filesResolved.begin();
             while (it != filesResolved.end()) {
-                const std::string& name = it->first;
+                const std::string& name = it->path();
                 // TODO: log if duplicated files were dropped
-                filesResolved.erase(std::remove_if(std::next(it), filesResolved.end(), [&](const std::pair<std::string, std::size_t>& entry) {
-                    return entry.first == name;
+                filesResolved.erase(std::remove_if(std::next(it), filesResolved.end(), [&](const decltype(filesResolved)::value_type& entry) {
```

Comment:
can we write `FileWithDetails` instead of `decltype(filesResolved)::value_type` ?

Reply:
Will do.

Reply:
Done.